tgui/packages/tgui/interfaces/CameraViewer.tsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * @file
+ * @copyright 2025
+ * @author glowbold (https://github.com/pgmzeta)
+ * @license MIT
+ */
+
+import { Button, ByondUi, Flex, Section, Stack } from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+import { capitalize } from './common/stringUtils';
+import { MinimapControllerData, MinimapMarkerData } from './MinimapController';
+
+export const CameraViewer = () => {
+  const { data, act } = useBackend<MinimapControllerData>();
+  const { title, theme, minimap_id, minimap_markers } = data;
+
+  return (
+    <Window title={title} theme={theme} width={950} height={700}>
+      <Window.Content>
+        <Stack justify="center">
```

Comment:
```suggestion
        <Stack>
```
If I'm reading correctly, justify shouldn't be needed.